website/partials/demo-thumbnail.pug

```diff
@@ -0,0 +1,18 @@
+include authors.pug
+
+mixin demo-thumbnail(source, title, authors, caption, disabled)
+  div(class = disabled ? 'disabled text-muted' : '').card
+    if disabled
+      img(src = 'img/' + source + '.jpg').img-fluid
+      h4.badge-parent #[span.badge.badge-dark in progress]
+    else
+      a(href = '/demos/' + source + '.html').card-img-top #[img(src = 'img/' + source + '.jpg').img-fluid]
```

Comment:
Do we really need the PNG versions of all images when only the JPG is used?

Reply:
hmm. where to store them? i like to have the originals nearby.